docker/scripts/cron/application_restart

```diff
@@ -1 +1 @@
-0 2 * * 0  root python3 start.py test --all_analyzers down && python3 start.py test --all_analyzers up -d
\ No newline at end of file
+0 2 * * 0  root ./start test down --all_analyzers && ./start test up --all_analyzers -- -d
```

Comment:
I meant this comment that I thought to have submitted:

I think that this cron has never worked properly. Can we assume an absolute working directory (something like `/opt/IntelOwl`) and launch the start.py from there and then add this consideration in the related documentation? Ty

Reply:
I should have applied your suggestion. Also updated doc.